src/main/java/com/okta/sdk/framework/JsonApiClient.java

```diff
@@ -55,10 +57,15 @@ public void serialize(DateTime value, JsonGenerator jsonGenerator, SerializerPro
             return null;
         }
         InputStream inputStream = response.getEntity().getContent();
-        JsonParser parser = objectMapper.getFactory().createParser(inputStream);
-        T toReturn = parser.readValueAs(clazz);
-        EntityUtils.consume(response.getEntity());
-        return toReturn;
+        try {
+            JsonParser parser = objectMapper.getFactory().createParser(inputStream);
+            T toReturn = parser.readValueAs(clazz);
+            EntityUtils.consume(response.getEntity());
+            return toReturn;
+        } catch (JsonParseException ex) {
+            throw new SdkException("Unable to process JSON: "
```

Comment:
I think I tried adding this here before and ran into issues because you can only read an entity once.  Please make sure you test locally with this to make sure you are logging an invalid json response correctly.